docs/setup/installation.md

```diff
@@ -2,50 +2,87 @@
 title: Installation
 ---
 
-SourceCred currently support running from a Node environment or using Docker.
-_In a future release we're looking to add easier options._
+The easiest way to install SourceCred is to fork the [example repo](https://github.com/sourcecred/example-instance). This repo contains scripts that will install SourceCred locally and set up an example instance of SourceCred. This instance can then be configured to include just the plugins you want and point to the data sources you care about.
 
-## Node environment
+If you want to build SourceCred from source (for instance to get bleeding edge features) or to develop on it, see the [Development Setup instructions](#development-setup).
 
-The Node environment is useful for development and running from source code.
+You can also run SourceCred from a [Docker image](#using-docker)  (useful for automated setups and canonical instances). 
 
-Make sure you've installed:
-- Node version 10 or 12.
-- Yarn version 1.7 or greater.
+## Installing from example instance
 
-Releases can be downloaded from https://github.com/sourcecred/sourcecred/releases.
-Or you can download the latest `master` branch if you're feeling brave.
+1. Clone the `example-instance` repo from SourceCred’s GitHub:
 
-Then run the following commands to download dependencies and compile the CLI
-(backend).
+`git clone https://github.com/sourcecred/example-instance.git`
 
-```bash
-cd /path/to/extracted/code
-yarn install
-yarn backend
-```
+2. Install SourceCred and its dependencies
 
-From here you can use the CLI.
+If you don't have it, install [yarn](https://classic.yarnpkg.com/lang/en/). Then run yarn to install SourceCred and its dependencies.
 
-```bash
-node bin/sourcecred.js help
+`yarn` 
+
+That's it! The installed instance with come with the latest stable release and default configurations, including all plugins.
+
+
+## Development Setup
```

Comment:
Let's not include dev setup here. It will be too annoying to try to keep it in sync between the official docs and the sc/sc repo. Instead, let's just link to the README.md on sourcecred/sourcecred for folks who want dev instructions.

Reply:
Added link to the README. We may want to tweak text. Didn't want to spend time on it because we may end up shifting things around, per other comments.